src/shared/codelens/typescriptCodeLensProvider.ts

```diff
@@ -30,7 +30,12 @@ import {
     getRuntimeForLambda,
     LambdaLocalInvokeParams,
     LocalLambdaRunner,
- } from './localLambdaRunner'
+} from './localLambdaRunner'
+
+const unsupportedNodeJsRuntimes: Set<string> = new Set<string>([
+    'nodejs4.3',
+    'nodejs6.10',
```

Comment:
Double-check that `nodejs` equals `nodejs8.10` and not `nodejs0.10` (see https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html )?

Reply:
I think sam cli uses nodejs8.10 as the default. 
When you run `sam init --runtime nodejs` it produces a nodejs8.10 project.
It takes the first config in the entry, [at this time](https://github.com/awslabs/aws-sam-cli/blob/develop/samcli/local/common/runtime_template.py#L33) it is nodejs8.10